libs/tokens/src/state/tokenLists/tokenListsStateAtom.ts

```diff
@@ -59,7 +59,7 @@ export const allListsSourcesAtom = atom((get) => {
 
 // Lists states
 export const listsStatesByChainAtom = atomWithStorage<TokenListsByChainState>(
-  'allTokenListsInfoAtom:v4',
+  'allTokenListsInfoAtom:v5',
```

Comment:
Any migration?
This wipes out user's info! Can make people unhappy

Reply:
<img width="977" alt="image" src="https://github.com/user-attachments/assets/310b9b4f-5bba-40c0-a991-f5223d66cd1a" />

Emergency JS takes care of it.
There is no users data in this atom